worlds/ff6wc/__init__.py

```diff
@@ -369,6 +376,67 @@ def create_items(self) -> None:
             self.create_event("Victory"))
         self.multiworld.completion_condition[self.player] = lambda state: state.has("Victory", self.player)
 
+        # Check to see what the Item Rewards are to populate the "dead" checks
+        # NOTE: most of this code is located in WorldsCollide/args/items.py during the process function
+        self.item_rewards = []
+        self.item_nonrewards = []
+        # if -ir in flagstring, then user specified item rewards
+        if "-ir" in self.options.Flagstring.value.split(" "):
```

Comment:
I wonder if this whole 50-line section could be extracted out into a function that we import from a different file, to keep things cleaner here.

It looks like maybe we could pass `options` as a parameter and just return `item_rewards`?

Reply:
Yeah I think that section just needs various different things in self.options.Flagstring to determine what can & cannot be in item_rewards.